batching_kafka_consumer/__init__.py

```diff
@@ -101,14 +101,16 @@ def __init__(self, topics, worker, max_batch_size, max_batch_time, bootstrap_ser
         self.worker = worker
 
         self.max_batch_size = max_batch_size
-        self.max_batch_time = max_batch_time
+        self.max_batch_time = max_batch_time  # in milliseconds
```

Comment:
Would like to rename this `max_batch_time_ms`, but probably makes sense to wait until there other breaking API changes.